gitdm/models.py

```
@@ -74,28 +83,98 @@
+        related_name="doctor_profile",
+    )
+    medical_code = models.CharField(max_length=32, blank=True, db_index=True)
+    specialty = models.CharField(max_length=120, blank=True)
+
+    class Meta:
+        verbose_name = "Doctor Profile"
+        verbose_name_plural = "Doctor Profiles"
+        indexes = [
+            models.Index(fields=["medical_code"]),
+            models.Index(fields=["specialty"]),
+        ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)